### PR TITLE
docs(projects): remove blocked link

### LINKED ISSUE #99: Repository link is broken.
### Greetings.
On the [Projects](https://ideashub.netlify.app/projects) page, the Currency Converter [repository link](https://github.com/Roshaen/currency-converter) is not working. It's maybe because the author has removed that repository from his profile. <br>
Consider fixing that link!

### PR DESCRIPTION
**Pull request by numanzamandipuu**
## Description
A link to project repository called _Currency Converter_ from https://ideashub.netlify.app/projects is invalid. Either the author has removed that repo from his profile, or for some reason, that link is not working. <br>
image
<br>
## Context
Original link - https://github.com/Roshaen/currency-converter
Fixed link - https://github.com/Roshaen/Roshaen
The new link takes you directly to the author's profile README.md because I didn't find any suitable repository to show.
## Related Issue
This PR fixes #99 
## How Has This Been Tested?
I've tested new projects page using [netlify.app](https://netlify.app/) and the testrun is on [this link](https://ideahub-test.netlify.app/projects).

### CODE REVIEW

File: data/projects.json [hunk 1]
```diff
@@ -30,13 +30,13 @@
...
+        "name": "Roshaen",
         "display": "Currency Convertor",
         "description": "A simple website using HTML, CSS, Javascript that converts one currency amount to other",
         "author": "Roshaen",
         "category": "website",
         "level": "basic",
-        "site": "https://roshaen.github.io/currency",
+        "site": "https://github.com/Roshaen",
         "tags": "html, css, js"
     },
```
Comment (ming-tsai) [inline data/projects.json:41]: Could you help me to remove this data?

File: docs/WEBSITE_BASIC.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Website - Basic
-  - [Currency Convertor](https://github.com/Roshaen/currency-converter) - [@Roshaen](https://github.com/Roshaen)
+  - [Currency Convertor](https://github.com/Roshaen) - [@Roshaen](https://github.com/Roshaen)
```
Comment (ming-tsai) [inline docs/WEBSITE_BASIC.md:2]: Please rollback this file change, due to there is a bot that will update this file.

**Review by ming-tsai — COMMENTED** [review]
Hi @numanzamandipuu,
I have some suggestions for you.

File: data/projects.json (hunk 1 above)
Comment (numanzamandipuu) [reply data/projects.json]: Hi @ming-tsai
I've removed lines 32-41 on [this](https://github.com/MakeContributions/ideahub/pull/100/commits/99614bb4bd3fc1a98f40ab3b8a4a00860453b460) commit. Is it okay?

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**